sources/packages/web/src/assets/css/base.scss

```diff
@@ -137,20 +136,31 @@ ul.no-bullets {
 
 // horizontal divider width
 .horizontal-divider {
-  border: 1px solid #E8E8E8;
-} 
+  border: 1px solid #e8e8e8;
```

Comment:
I can see that this is just a "style" format change and I believe that mine is configured is the same way.
@ann-aot, @guru-aot and @henryoforeh-aot, is anyone using a different configuration?

Reply:
mine is 
![image](https://user-images.githubusercontent.com/77353155/153086921-ff6d7291-8e75-4a83-9967-5b60228f8036.png)

Reply:
@guru-aot and @henryoforeh-aot, @ann-aot and I tested and we are with the same configuration tht @dheepak-aot is using. Please check your scss styles if the formatting is being applied in the same way as @dheepak-aot code in this PR.